test/cmd/README.md

```diff
@@ -0,0 +1,196 @@
+# OpenShift Command-Line Integration Test Suite
+
+This document desceribes how OpenShift developers should interact with the OpenShift command-line integration test suite.
+
+## Test Strucure
+
+The script to run the entire suite lives in [`hack/test-cmd.sh`](./../../hack/test-cmd.sh). All of the test suites that make up
+the parent suite live in `test/cmd`, and are divided by functional area. 
+
+## Running Tests
+
+The full test suite can be invoked by running any of the following `make` targets, which run this suite among others,
+or the test runner script itself:
+```sh
+$ hack/test-cmd.sh
+$ make check-test
+$ make test-int-plus
+```
+
+Furthermore, any singular test suite can be run if an OpenShift instance is running and you are the cluster admin by invoking
+any of the scripts inside of `test/cmd`. The scripts will create a project for themselves to run inside of, and delete it after
```

Comment:
Hm. I think it is. Should we change that? I remember @smarterclayton saying that ideally these should be runnable from a running OpenShift instance and that they would do everything necessary to run themselves